packages/blade/src/components/Checkbox/__tests__/__snapshots__/CheckboxGroup.web.test.tsx.snap

```diff
@@ -225,11 +225,12 @@ exports[`<CheckboxGroup /> should render with help text 1`] = `
           class="c2"
           display="flex"
         >
-          <div
-            class="c3"
-            color="surface.text.subtle.lowContrast"
+          <p
+            class="c3 Text__StyledText-wkh56f-0"
+            color="surface.text.subdued.lowContrast"
```

Comment:
this why it changed @chaitanyadeorukhkar 
https://github.com/razorpay/blade/pull/602/files#r957169857